src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV2.cs

```diff
@@ -38,7 +39,7 @@ protected static class ArtifactEventProperties
         {
             public static readonly string BuildType = "buildType";
             public static readonly string Project = "project";
-            public static readonly string BuildPipelineDefinition = "definition";
+            public static readonly string PipelineDefinition = "pipeline";
```

Comment:
We are changing input again? 🤞 
will this break old task vs. agent contains this change?

Reply:
yes, we are changing it.. The RM team is aware of the change and right now they won't be using the V2 version. They will be only using V1 version until this new task is fully tested on their end.

Reply:
can customer pick v2 version themselves? if so, they will break?

Reply:
The can pick V2 task but what broke the RM was auto-injection right? As long as RM doesn't auto-inject V2 task, it should be fine? Let me know if we are missing something here. Thanks!

Reply:
We've explicitly asked them to specify the major version for auto injection.